rueppellii-lvlup-asp.net-core/Controllers/BadgesController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using rueppellii_lvlup_asp.net_core.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace rueppellii_lvlup_asp.net_core.Controllers
+{
+  [ApiController]
+  [Route("[controller]")]
```

Comment:
this is not equal to the route specified
http://jira.greenfox.academy/projects/RUELVLA/issues/RUELVLA-4